packages/spark-core/base/inputs/_text-input.scss

```diff
@@ -0,0 +1,97 @@
+// ==================================================================
+// Input Component
+// ==================================================================
+.sprk-b-TextInput {
+  font-size: $font-size-body-two;
+  font-weight: $font-weight-body-two;
+  line-height: $line-height-body-two;
+  outline: 0;
+  padding: 3px $space-s;
+  box-shadow: $input-box-shadow;
+}
+
+.sprk-b-TextInput[disabled] {
+  background-color: $gray-light;
+  box-shadow: $input-disabled-box-shadow;
+  color: $input-gray-medium;
+}
+
+.sprk-b-TextInput--date-picker,
+.sprk-b-TextInput--monetary {
+  position: relative;
+}
+
+.sprk-b-DatePicker__icon,
+.sprk-b-TextInput--monetary::before {
+  position: absolute;
+  z-index: $layer-height-xs;
+}
+
+.sprk-b-DatePicker__icon {
```

Comment:
It's not related to this line but wanted to note that on the iPhone, when you click into the datepicker field (regardless of how far up or down the datepicker is on your screen) It opens the datepicker and also opens the keyboard which flys up and you cant see the datepicker to pick a date. If you close the keyboard to pick a date then the datepicker closes. So, you cant pick a date on iPhone.

Reply:
i noticed the "copy" code button is covering the datepicker as well. I have an issue out that fixes this for modals by adding a higher z-index for modals in the spark site. Thinking maybe we need to do that for datepickers in the spark site or (better yet) change the elevation of the copy code button in the spark site.

Reply:
hmm, might be best to disable the picker on mobile, the field functions as a normal date field anyway.

Reply:
The zindex system we're using now puts all drizzle stuff higher than spark component stuff. That might not be the best approach. I'll take a look, will likely fix in a separate PR though.

Reply:
will address datepicker changes in another pr